### ISSUE TITLE
Plans for other mice?

### ISSUE THREAD

**Issue by duplexsystem** [pop-os/mouse-configurator#19]
Are there plans for other HP mice? How about non HP ones or will this remain HP mouse exclusive? It would be cool if this could be expanded to be a front end for libratbag as well.

**Comment by ids1024** [comment]
I don't think it would be too complicated to support other HP mice using the same protocol. The way things are implemented is fairly closely tied to the way that protocol works so it wouldn't easily accommodate other brands.
